<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QFrame" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="QWidget_1" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLineEdit" name="url_input">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>670</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding:5px</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameUrl_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Game URL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="set_url">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>730</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Get Game Data</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="QWidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>800</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="game_logo">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>55</y>
      <width>190</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="game_name">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Game Name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="game_id">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Game ID :</string>
    </property>
   </widget>
   <widget class="QLabel" name="game_plays">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Game Plays :</string>
    </property>
   </widget>
   <widget class="QLabel" name="game_likes">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Game Likes :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="game_name_value">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="game_id_value">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="game_plays_value">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="game_likes_value">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="game_description">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Game Description :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="game_descriptio_value">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>500</width>
      <height>150</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Time :</string>
    </property>
   </widget>
   <widget class="QLabel" name="game_description_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>New Score :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="score_value">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>600</y>
      <width>250</width>
      <height>50</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="update_score">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>685</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Update Score</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>740</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="time_value">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="border">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>border</zorder>
   <zorder>game_name</zorder>
   <zorder>game_id</zorder>
   <zorder>game_plays</zorder>
   <zorder>game_likes</zorder>
   <zorder>game_name_value</zorder>
   <zorder>game_id_value</zorder>
   <zorder>game_plays_value</zorder>
   <zorder>game_likes_value</zorder>
   <zorder>game_description</zorder>
   <zorder>game_descriptio_value</zorder>
   <zorder>time</zorder>
   <zorder>game_description_3</zorder>
   <zorder>score_value</zorder>
   <zorder>update_score</zorder>
   <zorder>back_button</zorder>
   <zorder>time_value</zorder>
   <zorder>game_logo</zorder>
  </widget>
  <action name="actionDsafasf">
   <property name="text">
    <string>dsafasf</string>
   </property>
  </action>
  <zorder>QWidget_2</zorder>
  <zorder>QWidget_1</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
